src/components/OpenStreetMap.js

```diff
@@ -35,8 +36,13 @@ const OpenStreetMap = (props) => {
 		height = 300,
 		latitude,
 		longitude,
+		instanceId = '',
 	} = props;
+
 	const [Leaflet, setLeaflet] = useState(null);
+	const mapId = `map-${instanceId || Math.random().toString(36).substr(2, 9)}`;
```

Comment:
Can use `uuid` here. Example: https://github.com/GatherPress/gatherpress/blob/develop/src/blocks/dropdown/edit.js#L85

Reply:
@mauteri Done. And fixed a couple other things